src/main/scala/io/qbeast/spark/delta/OTreeIndex.scala

```diff
@@ -81,9 +83,18 @@ case class OTreeIndex(index: TahoeLogFileIndex) extends FileIndex {
     }.toArray
     val stagingStats = stagingFiles
     val fileStats = qbeastFileStats ++ stagingStats
-
+    val sc = SparkSession.active.sparkContext
```

Comment:
Instead of getting the global spark session, which could give some issues if we have multiple spark session (e.g. in testing), you can use the spark session that is in the TahoeLogFileIndex
```scala
val sc = index.spark.sparkContext
```